doc/api.rst

```diff
@@ -33,6 +33,13 @@ This page lists all available functions and classes of `skrub`.
 
    Joiner
 
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+   :nosignatures:
+
+   InterpolationJoin
```

Comment:
Does not appear in the API generated docs (because you renamed it in the meantime)
```suggestion
   InterpolationJoiner
```